### PR TITLE
ADD new file

### PR DESCRIPTION
**Pull request by nikunj-russmediatech**

### CODE REVIEW

File: file2.txt [hunk 1]
```diff
@@ -1 +1,4 @@
 THIS IS TEST 2 FILE FROM FORK REPO
+
```
Comment (sanjay-russmediatech) [inline file2.txt]: THIS IS REVIEW

Comment (sanjay-russmediatech) [inline file2.txt]: TEST @nikunj-russmediatech

**Review by sanjay-russmediatech — COMMENTED** [review]
@nikunj-russmediatech plz check

File: file2.txt (hunk 1 above)
Comment (nikunj-russmediatech) [reply file2.txt]: Working on that.

Comment (sanjay-russmediatech) [reply file2.txt]: FINISH

**Pull request merged**